app/templates/index.html

```diff
@@ -4,45 +4,44 @@
 <head>
     <meta charset="UTF-8">
     <title>Wyze-Bridge</title>
-    <link rel="stylesheet" type="text/css" href="{{ url_for('static',filename='site.css') }}" />
-    {% if show_video %}
+    <link rel="stylesheet" type="text/css"
+        href="{{ 'static/site.css' if hass else url_for('static',filename='site.css') }}" />
```

Comment:
Unfortunately url_for static seems to build the hrefs with a leading forward slash which breaks in HASS since the static files are located somewhere like: `http://homeassistant.local:8123/api/hassio_ingress/FX8RhC_HKiSPRVURubziW8OCqGIxj-7s5kMt3L7RObU/static/site.css`
and it tries to grab the static files from `homeassistant.local:8123/static/`